README.md

```diff
@@ -1,100 +1,137 @@
+   # Required before the first startup (or if you've made updates to KB sources)
+   docker compose run machine build
+   # start platform services and run web interface at localhost:8000
+   docker compose up
+   ```
 
-To see and interact you with system run the following script for interface:
-```sh
-cd ostis-web-platform/scripts/
-./run_scweb.sh
-```
++ Native installation
+
+   ```sh
+   # Build knowledge base
+   cd ostis-web-platform/scripts/
+   ./build_kb.sh
+   # Launch knowledge processing machine
+   cd ostis-web-platform/scripts/
+   ./run_sc_server.sh
+   # *in another terminal*
+   # Launch semantic web interface at localhost:8000
+   cd ostis-web-platform/scripts/
+   ./run_scweb.sh
+   ```
+
+## Documentation
+
+We document all information about project development and its components' implementation in sources of its knowledge base,
+to provide opportunity to use it in information processing and knowledge generation.
+
+You can access the current version of the documentation in [docs/main.pdf](docs/main.pdf) file of this project.
 
-# Documentation on SC-code representation
-We represent all information about project development and its components' implementation in sources of its knowledge 
-base to provide opportunity to use it in information processing and knowledge creating. Documentation is written with 
-the help of LaTex tools in SCn-code representation. To see more information about [the OSTIS Technology](https://github.com/ostis-ai/ostis-standard) 
-web-oriented platform implementation:
+Documentation is written with
+the help of LaTeX tools in SCn-code representation. To build documentation manually, you'll need a LaTeX distribution installed on your computer. Alternatively we're providing a Docker image to build the documentation in case you can't / don't want to install LaTeX on your PC.
 
-We're providing a Docker image to build the documentation.
+### Download subprojects documentation
 
-To do that, run docker image:
 ```sh
-docker run -v <path/to/ostis-web-platform/>:/workdir ostis/scntex-builder 'cd docs && pdflatex -interaction=nonstopmode main.tex'
+# feel free to skip this step if the platform is already installed natively
+cd ostis-web-platform/scripts
+./prepare.sh no_build_sc_machine no_build_sc_web
 ```
 
-After the compilation, the `main.pdf` file should appear at `ostis-web-platform/docs/`.
++ ### Build steps (using LaTeX)
+
+   ```sh
+   cd ostis-web-platform/docs
+   latexmk -pdf -bibtex main.tex
+   ```
+
++ ### Build steps (using Docker)
+
+   ```sh
+   cd ostis-web-platform
+   docker run -v ${PWD}:/workdir -w /workdir/docs --rm -it ostis/scn-latex-plugin:latest "main.tex"
+   ```
+
+   After the compilation, the `main.pdf` file should appear at `ostis-web-platform/docs/`.
+
+## Feedback
+
+Contributions, bug reports and feature requests are welcome! Feel free to check our [issues page](https://github.com/ostis-ai/ostis-web-platform/issues) and file a new issue (or comment in existing ones).
```

Comment:
```suggestion
Contributions, bug reports, and feature requests are welcome! Feel free to check our [issues page](https://github.com/ostis-ai/ostis-web-platform/issues) and file a new issue (or comment in existing ones).
```

Reply:
Oxford comma is a bit disorienting for a Russian-speaking auditory, so I think it's better to skip it.